genon/preprocessor/facade/intelligent_processor.py

```
@@ -2479,6 +2618,27 @@ def check_glyphs(self, document: DoclingDocument) -> bool:
         return False
 
+    def check_empty_text(self, document: DoclingDocument) -> bool:
+        """텍스트 클러스터(박스)는 있는데 그 텍스트가 전부 비어 있는 페이지가 있는지 확인.
+
+        length 폴백(layout_only)이나 텍스트레이어 부재 등으로 박스만 있고 텍스트가
+        안 채워진 페이지를 잡아 강제 OCR 로 보낸다(이슈 #278 B-2).
+        """
+        from collections import defaultdict
+        page_item_count: dict = defaultdict(int)
+        page_text_len: dict = defaultdict(int)
+        for item, level in document.iterate_items():
```

Ruff (0.15.18)

[warning] Loop control variable `level` not used within loop body

Rename unused `level` to `_level`

(B007)